lib/ship.rb

```diff
@@ -0,0 +1,25 @@
+require 'pry'
+
+class Ship
+  attr_reader :name, :length, :health
+
+  def initialize(name, length)
+    @name = name
+    @length = length
+    @health = length
+    @sunk = false
```

Comment:
You mentioned we might not need this @sunk...?

Reply:
Oh you're right, let me update before we merge.